### PR TITLE
#1886 - Switch to MariaDB and update admin guide

### PR DESCRIPTION
**Pull request by UWinch**
**What's in the PR**
* update documentation on mysql server regarding unicode
* update section on apache reverse proxy for newer apache versions
* minor changes e.g. typos
* changes documentation to MariaDB
* updates Docker Compose for MariaDB

**How to test manually**
* Build the documentation and read the relevant sections of the admin guide

**Documentation**
* [x] PR updates documentation

### CODE REVIEW

File: inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_database_mysql.adoc [hunk 1]
```diff
@@ -28,21 +28,40 @@ the application.
 ----
 $ apt-get install mysql-server
 ----
-* make sure your MySQL server is configured for UTF-8. Check the following line is present in `/etc/mysql/mariadb.conf.d/50-server.cnf` (this is specific to Debian 9; on other systems the relevant file
-may be `/etc/mysql/my.cnf`):
+
+[[character-set-config]]
+* make sure your MySQL server is configured for 4 Byte UTF-8 (utf8mb4) to ensure that all unicode characters can be represented (e.g. emojis). In MySQL versions 8 and upwards, this is the default. Check that `character-set-server`, `default-character-set` are set to  `utf8mb4` and `collation-server` is set to `utf8mb4_unicode_ci` by logging into MySQL with
```
Comment (jcklie) [inline inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_database_mysql.adoc:33]: In newer versions of mysql, it is (utf8mb4, utf8mb4_0900_ai_ci) so I would write either (utf8mb4, utf8mb4_0900_ai_ci) or (utf8mb4, utf8mb4_unicode_ci). Also, it is dangerous to change collation and charset so I would add a warning.

Comment (UWinch) [reply inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_database_mysql.adoc:33]: How is it dangerous? I would assume only when the database was already running and you would risk corrupting the content. is it also dangerous when setting up for the first time?

Comment (jcklie) [reply inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_database_mysql.adoc:33]: If you have no data in there then it is not risky

Comment (UWinch) [reply inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_database_mysql.adoc]: Thanks, I made the changes.

**Comment by UWinch** [comment]
Jenkins, can you test this please?

**Comment by reckart** [comment]
How about switching the DB to MariaDB while we are at it?

**Pull request merged**